Makefile

```diff
@@ -140,13 +138,15 @@ _out/golangci-lint: outdir
 		echo "Using golangci-lint cached at $(GOLANGCI_LINT_BIN)";\
 	fi
 
-
 _out/git-semver: outdir
-	@go build -o _out/git-semver vendor/github.com/mdomke/git-semver/main.go
+	@GOBIN=$(shell pwd)/_out go install github.com/mdomke/git-semver/v6@latest
```

Comment:
floating tags created a lot of pain in the past. Let's use `v6@v6.9.0` or any fixed, non-floating tag you prefer.

Reply:
fixed! :)